client/rest-high-level/src/main/java/org/elasticsearch/client/MachineLearningClient.java

```diff
@@ -1877,4 +1882,129 @@ public void setUpgradeModeAsync(SetUpgradeModeRequest request, RequestOptions op
             listener,
             Collections.emptySet());
     }
+
+    /**
+     * Create a new data frame analytics config
+     * <p>
+     * For additional info
+     * see <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-put-calendar.html">
```

Comment:
Wrong link and not a calendar

Reply:
The documentation for these endpoints isn't up yet. See https://github.com/elastic/elasticsearch/blob/feature-ml-data-frame-analytics/x-pack/plugin/src/test/resources/rest-api-spec/api/ml.get_data_frame_analytics.json which has no 'documentation' field. 

Make the links TODOs for now and we can revisit this

Reply:
Done.